earthly/rust/stdcfgs/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.75.0"
+channel = "1.77.2"
```

Comment:
Shouldn't we use the latest stable version `1.78.0`?

Reply:
@stevenj Is there a reason why we are still using 1.77?

Reply:
We are now on 1.78 everywhere